MiniApp/Classes/core/MiniAppInfoFetcher.swift

```diff
@@ -41,4 +41,39 @@ internal class MiniAppInfoFetcher {
             }
         }
     }
+
+    func getMiniAppMetaInfo(miniAppId: String, miniAppVersion: String, apiClient: MiniAppClient, completionHandler: @escaping (Result<MiniAppManifest, Error>) -> Void) {
+
+        apiClient.getMiniAppMetaData(appId: miniAppId, versionId: miniAppVersion) { (result) in
+            switch result {
+            case .success(let responseData):
+                if let decodeResponse = ResponseDecoder.decode(decodeType: MetaDataResponse.self,
+                                                               data: responseData.data) {
+                    return completionHandler(.success(self.prepareMiniAppManifest(
+                                                        metaDataResponse: decodeResponse)))
+                }
+                return completionHandler(.failure(NSError.invalidResponseData()))
+            case .failure(let error):
+                return completionHandler(.failure(error))
+            }
+        }
+    }
+
+    func prepareMiniAppManifest(metaDataResponse: MetaDataResponse) -> MiniAppManifest {
```

Comment:
Couldn't this method be `private`?

Reply:
Done

Reply:
The class is internal already. Do we need to make it private? Because this method is also used in Helpers

Reply:
I think it's good to keep it internal